packages/flutter_version_resolver/bin/flutter_version_resolver.dart

```diff
@@ -0,0 +1,43 @@
+import 'dart:io';
+
+import 'package:flutter_version_resolver/flutter_version_resolver.dart';
+import 'package:flutter_version_resolver/src/logger.dart';
+import 'package:scoped_deps/scoped_deps.dart';
+
+/// Resolves the Flutter version for a package and optionally writes it to a
+/// file.
+///
+/// Usage:
+/// ```sh
+/// dart run bin/flutter_version_resolver.dart <path-to-package> [<output-file>]
+/// ```
+Future<void> main(List<String> arguments) async {
+  return runScoped(() async {
+    if (arguments.isEmpty || arguments.length > 2) {
+      logger.err(
+        'Usage: dart run bin/flutter_version_resolver.dart <path-to-package> [<output-file>]',
+      );
+      return;
+    }
+
+    final packageDirectory = Directory(arguments[0]);
+    if (!packageDirectory.existsSync()) {
+      logger.err(
+        'Package directory does not exist: ${packageDirectory.path}',
+      );
+      return;
```

Comment:
we should probably exit with a non-zero exit code in this case.

Reply:
Done